tools/codegen/dest/register_dispatch_key.py

```diff
@@ -60,30 +60,15 @@ def gen_create_out_helper(backend_index: BackendIndex) -> List[str]:
         dispatch = str(backend_index.dispatch_key).lower()
         empty_impl = f"at::detail::empty_{dispatch}"
         empty_strided_impl = f"at::detail::empty_strided_{dispatch}"
-        runtime_empty_supported_check = ""
-    elif backend_index.dispatch_key == DispatchKey.CompositeExplicitAutograd:
+    elif backend_index.dispatch_key in (
+            DispatchKey.CompositeExplicitAutograd, DispatchKey.QuantizedCPU, DispatchKey.QuantizedCUDA):
```

Comment:
tiny nit if we care about the perf (probably not necessary, just calling it out): we could avoid the dispatcher hop here from calling `at::empty`, but we'd have to abide by the naming convention and name the native kernels `at::native::empty_quantizedcpu/cuda`

Reply:
oh this used to not be easy to do but now it is easy